velox/core/QueryConfig.h

```diff
@@ -282,6 +291,10 @@ class QueryConfig {
     return get<int32_t>(kTestingSpillPct, 0);
   }
 
+  int32_t maxSpillLevel() const {
```

Comment:
Nice. Thank you for adding this configuration property. It would be nice to extract this change and corresponding logic in HashBuild into a separate PR.